### ISSUE TITLE
Error: 'message' when executing a streaming prompt

### ISSUE THREAD

**Issue by simonw** [taketwo/llm-ollama#9]
Got this just now:
```
llm -m mistral:latest '1 + 3'
```
> The sum of 1 and 3 is 4. So the answer to the expression "1 + 3" is 4.Error: 'message'

With a stack trace:
```bash
python -i $(which llm) -m mistral:latest '1 + 3'
```
```
 The sum of 1 and 3 is 4. So the answer to the expression 1 + 3 is 4.Error: 'message'
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniconda/base/lib/python3.10/site-packages/llm/cli.py", line 277, in prompt
    for chunk in response:
  File "/opt/homebrew/Caskroom/miniconda/base/lib/python3.10/site-packages/llm/models.py", line 91, in __iter__
    for chunk in self.model.execute(
  File "/opt/homebrew/Caskroom/miniconda/base/lib/python3.10/site-packages/llm_ollama.py", line 132, in execute
    yield chunk["message"]["content"]
KeyError: 'message'
```
This code here: https://github.com/taketwo/llm-ollama/blob/55bf5788c0c6a4761c2792cf716b036515232c66/llm_ollama.py#L124-L139

**Comment by simonw** [comment]
Running the alternative no-stream code path avoids the error:
```bash
llm -m mistral:latest '1 + 3' --no-stream
```
>  The sum of 1 and 3 is 4. So the answer to the expression "1 + 3" is 4.

**Comment by simonw** [comment]
Before I found your plugin I started work on my own, which used this pattern:
```python
import llm
import ollama


@llm.hookimpl
def register_models(register):
    for model in ollama.list()["models"]:
        register(OllamaModel(model["name"]))


class OllamaModel(llm.Model):
    can_stream = True

    def __init__(self, ollama_id):
        self.ollama_id = ollama_id
        self.model_id = "ollama:{}".format(ollama_id)

    def execute(self, prompt, stream, response, conversation):
        for chunk in ollama.chat(
            self.ollama_id,
            messages=[
                {
                    "role": "user",
                    "content": prompt.prompt,
                },
            ],
            stream=True,
        ):
            try:
                yield chunk["message"]["content"]
            except KeyError:
                pass
```

**Comment by taketwo** [comment]
Fixed by #10.
